app.py

```diff
@@ -26,8 +29,7 @@
 import hashlib
 
 # app.py
-from flask import Flask, request, jsonify, session, abort, g, redirect, make_response, Response
-from flask_helpers import render_template
+from flask import Flask, request, jsonify, render_template, session, abort, g, redirect, make_response, Response
```

Comment:
We should restore the two lines above and remove this one, this was a change recently introduced.

Reply:
Fixed!